src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Publisher.kt

```diff
@@ -25,12 +26,38 @@ import com.google.pubsub.v1.PubsubMessage
 import com.google.pubsub.v1.TopicName
 import org.slf4j.LoggerFactory
 
-fun getResultData(taskId: String, compiledBytes: ByteString, outputFileName: String, statusCode: Int): ByteString {
-    return CompilerFleet.CompilerFleetResult.newBuilder()
+
+fun buildResultData(taskId: String, compiledBytes: ByteString, statusCode: Int): ByteString {
+    val status = when (statusCode) {
+        0 -> CompileResponse.Status.OK
+        else -> CompileResponse.Status.FAILED
+    }
+
+    val identifier = CompilerFleet.RequestIdentifier.newBuilder()
             .setTaskId(taskId)
             .setResultBytes(compiledBytes)
```

Comment:
Just've noticed that there is PDF embedded into "identifier". Errrr.. what? There is a good place for PDF bytes in texbe response.

Reply:
Moved pdf to texbe response.